docs/cli.md

```diff
@@ -29,24 +29,25 @@ The general structure for running commands with the Gravitino CLI is `gcli entit
  ```bash
  usage: gcli [metalake|catalog|schema|table|column] [list|details|create|delete|update|set|remove|properties] [options]
  Options
- -a,--audit              display audit information
- -c,--comment <arg>      entity comment
- -g,--group <arg>        group name
- -h,--help               command help information
- -i,--ignore             ignore client/sever version check
- -l,--user <arg>         user name
- -m,--metalake <arg>     metalake name
- -n,--name <arg>         full entity name (dot separated)
- -P,--property <arg>     property name
- -p,--properties <arg>   property name/value pairs
- -r,--rename <arg>       new entity name
- -s,--server             Gravitino server version
- -t,--tag <arg>          tag name
- -u,--url <arg>          Gravitino URL (default: http://localhost:8090)
- -v,--version            Gravitino client version
- -V,--value <arg>        property value
- -z,--provider <arg>     provider one of hadoop, hive, mysql, postgres,
-                         iceberg, kafka
+ -a,--audit            display audit information
+ -c,--comment <arg>    entity comment
+ -f,--force            force operation
+ -g,--group <arg>      group name
+ -h,--help             command help information
+ -i,--ignore           ignore client/sever version check
+ -l,--user <arg>       user name
+ -m,--metalake <arg>   metalake name
+ -n,--name <arg>       full entity name (dot separated)
+ -P,--property <arg>   property name
+ -r,--role <arg>       role name
+    --rename <arg>     new entity name
```

Comment:
I think "-r" has no relationsip with "--rename" anymore, see the change in GravitinoOptions.java; if so, this line should be removed.

Reply:
This is correct output `--rename` doesn't use `-r` `--role` does. `--role` has no short argument shown.

Reply:
I see; Had thought that "-r" is the short-name for both "role" and "rename".